CHANGELOG.md

```diff
@@ -13,6 +13,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed: Optimize Solver by choosing actions in a smarter order. Prefer actions of types that are likely to progress th. Postpone dangerous actions. This should make the solver able to validate seeds where it previously timed out. Solving should in general be faster in general.
 - Fixed: Solver bug that made it unable to detect dangerous actions, which could result in some possible seeds being considered impossible.
 
+### Metroid Prime
+
+#### Logic Database
+
+##### Tallon Overworld 
+
+- Added: Biotech Research Area 1 - Easier gravityless NSJ method from room center to Deck Beta Security Hall (videos added)
+- Added: Root Cave - R-Jump method (video added)
+
+
+##### Phendrana Drifts
+- Added: Hunter Cave - Lower Edge Tunnel -> Hunter Cave Access, NSJ requires a slope jump or bomb jump after the grapple point to reach the platform with the doors.
+- Added: Hunter Cave - Hunter Cave Access -> Lower Edge Tunnell, NSJ requires an L-Jump to reach the platforms across the water without falling in. Added Gravity logic if falling in (matches Lake Tunnel -> Lower Edge Tunnel).
+- Added: Hunter Cave - Various videos
```

Comment:
No need to report videos being added for specific rooms. Just keep a running total of the number of new videos in the changelog under `#### Logic Database`

Reply:
For example: https://github.com/randovania/randovania/blob/main/CHANGELOG.md?plain=1#L137

Reply:
Understood, changes made in newest commit.